### PR TITLE
Update schema for order count from Vortex (GALL-1503)

### PR DESCRIPTION
**Pull request by starsirius**
https://artsyproduct.atlassian.net/browse/GALL-1503

This also updated v2 schema and want to make sure it's ok. cc @zephraph @alloy.

Closed the [previous one](https://github.com/artsy/metaphysics/pull/1734) and PRed from upstream branch.

### CODE REVIEW

File: _schemaV2.graphql [hunk 1]
```diff
@@ -9200,6 +9201,7 @@ type Query {
   # Find PartnerStats
   analyticsPartnerStats(partnerId: String!): AnalyticsPartnerStats
   marketingCollections(
+    category: String
```
Comment (iskounen) [inline _schemaV2.graphql]: This change seems unrelated to the order count field. Is it supposed to be included in this PR?

**Comment by alloy** [comment]
Yep, that’s expected, v2 is simply a transformed version of v1, also so that it stays up to date with in progress v1 work.

**Comment by iskounen** [comment]
Thanks for the clarification. For reference, this change was introduced to v1 in #1724.
